xchange-core/src/main/java/org/knowm/xchange/service/marketdata/MarketDataService.java

```diff
@@ -41,6 +42,25 @@ default Ticker getTicker(CurrencyPair currencyPair, Object... args) throws IOExc
     throw new NotYetImplementedForExchangeException("getTicker");
   }
 
+  /**
+   * Get a ticker representing the current exchange rate
+   *
+   * @return The Ticker, null if some sort of error occurred. Implementers should log the error.
+   * @throws ExchangeException - Indication that the exchange reported some kind of error with the
+   *     request or response
+   * @throws NotAvailableFromExchangeException - Indication that the exchange does not support the
+   *     requested function or data
+   * @throws NotYetImplementedForExchangeException - Indication that the exchange supports the
+   *     requested function or data, but it has not yet been implemented
+   * @throws IOException - Indication that a networking error occurred while fetching JSON data
+   */
+  default Ticker getTicker(Instrument instrument, Object... args) throws IOException {
+    if(instrument instanceof CurrencyPair) {
```

Comment:
not sure this is the best behaviour, ideally at some point I think we want to remove the curencyPair methods, so let's not make them dependant on each other.

Reply:
As a library user I'd like to have a method that works for any supported instrument in all exchanges. This approach:
1. provides such method
2. does not require to change all the exchanges
Otherwise we'll force users to implement wrappers with this logic.